### ISSUE TITLE
Temporal predicts with same probability for every video

### ISSUE THREAD

**Issue by hadarsh7798** [wushidonguc/Two-stream-action-recognition-keras#11]
When I try to train the temporal model with my custom dataset, the loss and accuracy values seems to oscillate a lot. Also when I use this to predict (currently only on two classes), it predicts the same class everytime for each video and and its every frame. I even tried a different architecture for the temporal stream. Even that didn't work.
